it/test/uk/gov/hmrc/tradergoodsprofiles/controllers/RemoveRecordControllerIntegrationSpec.scala

```diff
@@ -250,13 +228,14 @@ class RemoveRecordControllerIntegrationSpec
     }
 
     "return an BAD_REQUEST (400) if recordId is invalid" in {
-      stubForRouterBadRequest(
```

Comment:
I think this test may be redundant, as it just test that it return the error from the router. "return an error if the router returns an error" test does already that. I wonder if we miss the test that test of internal server error is thrown if w cannot perse the error response as json

Reply:
Yeah you're right the above test covers that. I'll put in the test around the internal server error